integration_guides/Networking/hpe-arubacx-dss.rst

```diff
@@ -0,0 +1,147 @@
+Aruba CX10000 Integration (ArubaCxDss Network Plugin)
+-----------------------------------------------------
+
+Overview
+^^^^^^^^
+The HPE ANW DSA (HPE Aruba Networking Distributed Service Architecture) plugin, also known as the ArubaCxDss Network Plugin, brings advanced networking features to HPE VME by leveraging Aruba CX10000 switches.
+It supports both micro-segmentation and macro-segmentation, integrating seamlessly with HPE and Aruba technologies to improve user experience and operational efficiency.
+
+Implemented as a |morpheus| ``NetworkProvider`` in Groovy, the plugin is purpose-built for the HPE ANW DSS Port Group network type, optimized for HPE VME environments.
+It automates the creation and management of networks for HPE ANW DSS Port Groups, connecting servers to Aruba CX10000 switches within a VME cluster.
+
+On each host, the plugin creates Linux bridge interfaces and associates them with VLANs for streamlined network management and configuration.
+|morpheus| integrates directly with AFC to automatically create or delete networks for HPE ANW DSS Port Groups as they are added to or removed from a configured cluster.
+Adding or removing hosts in the cluster triggers the corresponding network changes in AFC, ensuring that network configurations are always up-to-date.
+
+The plugin also enables the creation of networks with specific VLAN IDs, supporting flexible segmentation and isolation.
+When a network is created, the plugin provisions the required VLANs on the Aruba CX10000 switches, ensuring networks are ready for use by instances within the VME cluster.
+
+Additionally, the plugin creates a Libvirt network of type Private.
+When an instance is created on the host and associated with this network, a macvtap interface is generated and mapped to the corresponding Linux Interface VLAN ID, ensuring seamless connectivity and isolation.
+
+Features
+^^^^^^^^
+* Create and delete networks of type ``HPE ANW DSS Port Group`` directly from the integration.
+* Automatically create and delete networks in Aruba Fabric Composer (AFC) when hosts are added or removed from the cluster.
+* Manage network configurations through seamless integration with Aruba Fabric Composer (AFC).
+* Enable both micro-segmentation and macro-segmentation for flexible network isolation.
+* View detailed summaries and status of Aruba CX10000 switches.
+* Ensure automatic, real-time synchronization of network configurations with Aruba CX10000 switches.
+
+Prerequisites
+^^^^^^^^^^^^^
+The following requirements must be met to deploy and configure the ArubaCxDss Network plugin on VME.
```

Comment:
I think we should avoid using "ArubaCxDss".  Let's change it to "HPE Aruba CX DSS" throughout.  (Line 85 has the only other reference, after the changes in my other suggestions.)